services/integration-content-repository/k8s/deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: icr
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: icr
+    spec:
+      containers:
+        - name: icr
+          image: openintegrationhub/icr:latest
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 3001
+          env:
+            - name: MONGODB_URL
+              value: 'mongodb://icrdb.openintegrationhub.com'
+            - name: IAM_JWT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: oidc-oih-iam-dev
+                  key: 'jwtsecret'
+            - name: IAM_JWT_AUDIENCE
+              value: 'openintegrationhub.com'
+            - name: IAM_JWT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: oidc-oih-iam-dev
+                  key: 'jwtsecret'
+            - name: IAM_JWT_ISSUER
+              value: 'https://www.example.com'
+            - name: IAM_BASE_URL
+              value: 'https://iam.openintegrationhub.com'
+            - name: IAM_API_BASE_URL
+              value: 'https://iam.openintegrationhub.com/api/v1/'
+          livenessProbe:
+            httpGet:
+              path: /healthcheck
+              port: 3001
+            initialDelaySeconds: 120
+            timeoutSeconds: 1
+          readinessProbe:
+            httpGet:
+              path: /healthcheck
+              port: 3001
+            initialDelaySeconds: 10
+            timeoutSeconds: 1
```

Comment:
Nice to have / avoiding bottlenecks in the future: add "resources"/"requests"-limits such as cpu, memory (should be done in all services in the future), e.g: IAM/deployment/lines81-86

Reply:
done